Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+FROM centos:7.6.1810
+
+MAINTAINER darion.wang "darion.wang@vesoft.com"
+
+RUN yum -y install wget \
+                   openssh-clients \
+                   openssh-server net-tools
+
+RUN wget -O nebula-1.0.0-beta.el7-5.x86_64.rpm \ 
+    https://github.com/vesoft-inc/nebula/releases/download/v1.0.0-beta/nebula-1.0.0-beta.el7-5.x86_64.rpm
+
+RUN rpm -ivh nebula-1.0.0-beta.el7-5.x86_64.rpm
+
+RUN rm nebula-1.0.0-beta.el7-5.x86_64.rpm
```

Comment:
We should extract a `NEBULA_VERSION` env variable to avoid updating the package name each release.

Reply:
Good point

Reply:
each 'RUN' command will create a new layer